compiler/packages/babel-plugin-react-compiler/src/HIR/HIR.ts

```diff
@@ -1429,6 +1429,8 @@ export type ReactiveScope = {
   merged: Set<ScopeId>;
 
   loc: SourceLocation;
+
+  source: boolean;
```

Comment:
This name feels a bit too abbreviated to be immediately obvious what it means (when looking at a usage site in particular). Maybe `fromManualMemoization` or something descriptive?